File: QXQP-C# MVC/QXQPS/Vo/Bsgrid.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace QXQPS.Vo
{
    //T:泛型
    public class Bsgrid<T>
    {
        public bool success { get; set; }//成功否

        public int totalRows { get; set; }//总行数

        public int curPage { get; set; }//当前页面

        public List<T> data { get; set; }//数据
    }
}

File: QXQP-SSM/src/main/java/com/koi/util/Bsgrid.java
package com.koi.util;

import java.util.HashMap;
import java.util.List;
import java.util.Map;

import net.sf.json.JSONArray;
import net.sf.json.JSONObject;

public class Bsgrid {
	public static JSONObject getJson(JSONArray json,Long curPage,Long totalRows){
		JSONObject jsons=new JSONObject();
		jsons.put("data", json);
		jsons.put("curPage", curPage);
		jsons.put("totalRows", totalRows);
		jsons.put("success", true);
		return jsons;
	}
	public static <T> Map getList(List<T> list,Long curPage,Long totalRows){
		Map<String, Object> map = new HashMap<String, Object>();
		map.put("data", list);
		map.put("curPage", curPage);
		map.put("totalRows", totalRows);
		map.put("success", true);
		return map;
	}
}
